src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -89,13 +88,13 @@ public static Price parsePrice(String price) throws ParseException {
      *
      * @throws ParseException if the given {@code name} is invalid.
      */
-    public static BudgetName parseBudgetName(String budgetName) throws ParseException {
+    public static Name parseBudgetName(String budgetName) throws ParseException {
```

Comment:
Perhaps it is more appropriate if the argument "budgetName" is changed to "name"?